lib/widgets/action_sheet.dart

```diff
@@ -317,3 +319,51 @@ class CopyButton extends MessageActionSheetMenuItemButton {
       data: ClipboardData(text: rawContent));
   };
 }
+
+class StarButton extends MessageActionSheetMenuItemButton {
```

Comment:
Let's put the classes in the same order as the buttons appear in the UI (and in the list up in `showMessageActionSheet`. That will help when navigating the file.

Reply:
Also changed the order in the tests to match!